code-challenges.js

```diff
@@ -18,11 +39,28 @@ var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
 
+const whatRemains3 = (arr) => {
+    newArr = []
+    arr.map(value => {
+        if(typeof value == "number") {
+        newArr.push(value%3)
+        }
+    })
+    return newArr
+}
```

Comment:
good work! just remember with a map, you dont need to push into a new array since map returns a new array anyway :) . also, filter may work better in this situation. but the logic is all there!